src/server.ts

```diff
@@ -25,11 +24,17 @@ export async function createServer(
     }
 
     const redis = new Redis(serverConfig.REDIS_URL, { maxRetriesPerRequest: -1 })
-    redis.on('error', (error) => {
-        Sentry.captureException(error)
-        console.error('🔴 Redis error! Trying to reconnect.')
-        console.error(error)
-    })
+    redis
+        .on('error', (error) => {
+            Sentry.captureException(error)
+            console.error('🔴 Redis error encountered! Trying to reconnect...')
+            console.error(error)
+        })
+        .on('ready', () => {
+            if (!areWeTestingWithJest()) {
+                console.info(`✅ Connected to Redis at ${serverConfig.REDIS_URL}!`)
```

Comment:
I might be overanalysing this, but the `REDIS_URL` env often contains passwords and other sensitive data, which *might* not be nice to output into the log.